scripts/translationRunner.js

```diff
@@ -0,0 +1,8 @@
+// translationRunner.js
+const manageTranslations = require('react-intl-translations-manager').default;
+
+manageTranslations({
+  messagesDirectory: 'i18n/default/',
+  translationsDirectory: 'i18n/locales',
+  languages: ['fr'], // any translation---don't include the default language
+});
```

Comment:
Copied from https://mikebridge.github.io/articles/typescript-i18n-react-intl/, who copied it from https://github.com/GertjanReynaert/react-intl-translations-manager